packages/web_drivers/pubspec.yaml

```diff
@@ -2,11 +2,11 @@ name: web_driver_installer
 description: For downloading/installing different web drivers. Used in tests.
 
 environment:
-  sdk: ">=2.2.2 <3.0.0"
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  args: ^1.5.2
-  http: ^0.12.0
-  path: ^1.6.4
-  test: ^1.6.5
-  yaml: ^2.2.0
+  args: ">=1.5.2 <3.0.0"
+  http: ">=0.12.0 <1.0.0"
```

Comment:
This is not preventing you from picking up 0.13, 0.14... which are major versions according to pub. I think the upper limit here should be `<0.14.0`?

Reply:
We use a git dependency on this package from the engine, which uses tight constraints, so it won't let these version constraints go too far. Honestly, I'm tempted to make everything `any` here, just for simplicity.